hack/golangci-lint.sh

```diff
@@ -23,7 +23,7 @@ paths=""
 while IFS= read -r line; do
     # read directory from the file and append a wildcard
     paths+="${line}/... "
-done <hack/lint-paths.txt
+done <hack/linter/lint-paths.txt
 
 golangci-lint run --timeout 20m --verbose ${paths}
```

Comment:
Maybe something for a follow up but thinking about this some more do we want to move the config into this new `hack/linter` and symlink it back into the root of the project? That way the SIGs can approve changes to the config but it should still work for IDE integrations etc (I've just started using nvim-linter for example) that use the root config?

Reply:
https://github.com/kubevirt/kubevirt/pull/14154/commits/7f663a9db704f59559a2562c513535c53dda3f5c - I'll follow up with this next week.

Reply:
Yeah agreed and the PR keeps the root config in place, just sym-linked from `hack/linter` to allow for better approval control. I've updated https://github.com/kubevirt/kubevirt/pull/14154 but will hold it until we can discuss it in the next code quality meeting.